.github/workflows/tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.8]
```

Comment:
[3.6, 3.7, 3.8]  or just [3.8]?

Reply:
Since this is `lint` testing, I think 1 test is enough and naturally that should be the latest version of python.

But there is another reason for this which is that `importlib-metadata` recently broke `lint` tests on python 3.6 and 3.7 so rather than pinning it (which I previously did), I rather just test on the latest python version.